gpytorch/models/utils.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
Either move this to `__init__.py` in `gpytorch.utils` or define in `multi_output_gp.py` directly since that's where it gets used.

Reply:
Given that `_get_tuple_args` appeared directly in the likelihood, maybe let's just define it in `multi_output_gp.py` for now